index.js

```diff
@@ -492,21 +492,42 @@ async function sendInitialChatbotMessage(session) {
     locationData = await db.getLocationData(location)
```

Comment:
It's not part of this pull request, but since you're in the area, I notice that there are some indentation issues in the `sendTwilioMessage` function. Now might be a good time to clean that up.

Reply:
Cleaned.